tox.ini

```
@@ -52,8 +52,9 @@ deps=
     -rrequirements.dev.txt
     -rrequirements.txt
 commands_pre = mkdir -p coverage
-whitelist_externals = */mkdir
+allowlist_externals = */mkdir
+                      */bash
 setenv = MYPYPATH=stubs:src
 commands =
-    python -m mypy --soft-error-limit=-1 --junit-xml mypy.xml --cobertura-xml-report coverage -p inmanta
+    bash -c "python -m mypy --soft-error-limit=-1 --junit-xml mypy.xml --cobertura-xml-report coverage -p inmanta >> mypy_output.txt"
```

```suggestion
    sh -c "python -m mypy --soft-error-limit=-1 --junit-xml mypy.xml --cobertura-xml-report coverage -p inmanta > mypy_output.txt"
```
Nitpick, but we don't actually need bash, and we'd want to clear the file if it already exists.